src/modules/api.js

```diff
@@ -0,0 +1,32 @@
+import Scores from './scores.js';
+
+const display = (gameData) => {
+  const scoreDisplay = document.querySelector('.scores_name');
+  scoreDisplay.innerHTML = '';
+  // gameData.sort((a, b) => a.score - b.user);
+  const displayScores = gameData.map((list) => `<p class ='scoreList'>${list.user}: ${list.score} </p>`).join('');
+  scoreDisplay.innerHTML = displayScores;
+};
+
+const addScore = async (user, score) => {
+  const apiURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  const url = `${apiURL}games/`;
+  const ID = 'ajtuTgubydhYUNubdauu';
```

Comment:
This part of the code is repeated multiple times, you can export this to a new function or variable to make it reusable. So it's highly recommended to follow the **[DRY](https://codinglead.co/javascript/what-is-DRY-code)** principle.

Reply:
Thanks, I have made correction to this